app/uk/gov/hmrc/tariffclassificationfrontend/views/reject_case.scala.html

```diff
@@ -34,13 +34,11 @@ <h2 class="heading-large mt-0">Change the status of this case to: Rejected</h2>
 
             @helper.form(action = RejectCaseController.postRejectCase(c.reference),
             'enctype -> "multipart/form-data",
-            'id -> "reject-case") {
+            'id -> "reject_case") {
```

Comment:
Generally we've been doing ID in the form
`component_name-section_name-sub_section_name` 
i.e using `_` to replace spaces and `-` to separate sections

Reply:
That's fine, don't disagree. That ID has just been there like that since ancient times